girder/api/v1/resource.py

```diff
@@ -180,7 +180,15 @@ def _lookUpToken(self, token, parentType, parent):
         raise RestException('Child resource not found: %s(%s)->%s' % (
             parentType, parent.get('name', parent.get('_id')), token))
 
-    def _lookUpPath(self, path, user):
+    def _lookUpPath(self, path, user, test=False):
+        """
+        Look up a resource in the data hierarchy by path.
+        :param path: path of the resource
+        :param user: user with correct privledges to access path
+        :param test: defaults to false, when set to true
+        will return None instead of throwing exception when
+        path doesn't exist
```

Comment:
Sphinx will also complain if you don't indent continuations of these descriptions, e.g.

    :param blah: blah blah ...
        blah.